src/OrleansTestingHost/OrleansConfigurationForTesting.xml

```diff
@@ -17,6 +17,15 @@
        <!--
        <TraceLevelOverride LogPrefix="Runtime.Dispatcher" TraceLevel="Verbose" />
        <TraceLevelOverride LogPrefix="AssemblyLoader.Silo" TraceLevel="Warning" />
+       <TraceLevelOverride LogPrefix="Runtime.MembershipOracle" TraceLevel="Verbose" />
```

Comment:
Let's not clutter the file unless we want an exhaustive list of all prefixes here, which I don't think we want.